packages/account/src/Components/poi/status/unsupported/card-details/card-details.scss

```diff
@@ -5,7 +5,7 @@
     display: flex;
 
     &--mobile {
-        padding: 2.4rem 1.6rem;
```

Comment:
Can u check this again, in figma, the padding on right and left is 1.6rem
<img width="478" alt="Screenshot 2023-10-20 at 1 14 01 PM" src="https://github.com/binary-com/deriv-app/assets/98398322/31477668-045a-45d9-938a-bfe052f8cf44">

Reply:
it was initially 1.6rem which is too shallow and the card is to improve this. Design is based on the current production styles.

Reply:
This is how it looks like with 1.6rem padding on left & right
<img width="392" alt="Screenshot 2023-10-20 at 5 22 06 PM" src="https://github.com/binary-com/deriv-app/assets/129842155/f3b70539-73f2-4962-9e1d-609496e3e264">